### ISSUE TITLE
Latest version not starting: ValueError: 'AV' is not a valid ActionMode

### ISSUE THREAD

**Issue by valuta1995** [DasAmpharos/EonTimer#131]
```
Connected to pydev debugger (build 233.15026.15)
pygame 2.5.2 (SDL 2.28.3, Python 3.12.2)
Hello from the pygame community. https://www.pygame.org/contribute.html
Traceback (most recent call last):
  File "C:\Users\Username\AppData\Local\Programs\PyCharm Professional\plugins\python\helpers\pydev\pydevd.py", line 1534, in _exec
    pydev_imports.execfile(file, globals, locals)  # execute the script
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Username\AppData\Local\Programs\PyCharm Professional\plugins\python\helpers\pydev\_pydev_imps\_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "D:\Folder\EonTimer\EonTimer.py", line 32, in <module>
    sys.exit(main())
             ^^^^^^
  File "D:\Folder\EonTimer\EonTimer.py", line 23, in main
    context = AppContext(['eon_timer'], provided={QApplication: InstanceProvider(app)})
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Folder\EonTimer\eon_timer\util\injector\app_context.py", line 24, in __init__
    self.__get_or_create_provider(component_definitions, component_type)
  File "D:\Folder\EonTimer\eon_timer\util\injector\app_context.py", line 77, in __get_or_create_provider
    provider = self.__get_provider(scope, component_factory, dependency_providers)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Folder\EonTimer\eon_timer\util\injector\app_context.py", line 84, in __get_provider
    return SingletonProvider(factory, dependency_providers)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Folder\EonTimer\eon_timer\util\injector\provider.py", line 20, in __init__
    self.instance: Final = factory(*dependencies)
                           ^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Folder\EonTimer\eon_timer\util\properties\settings.py", line 24, in __init__
    self._read()
  File "D:\Folder\EonTimer\eon_timer\util\properties\settings.py", line 34, in _read
    prop.read(self.settings, name)
  File "D:\Folder\EonTimer\eon_timer\util\properties\property.py", line 105, in read
    self._value = self.enum_type(settings.value(name, initial_value))
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python312\Lib\enum.py", line 744, in __call__
    return cls.__new__(cls, value)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python312\Lib\enum.py", line 1158, in __new__
    raise ve_exc
ValueError: 'AV' is not a valid ActionMode
```

As far as I can tell, the default config has some issues?

**Comment by DasAmpharos** [comment]
Hey @valuta1995, thanks for opening an issue! I pushed a change that should fix this. Can you try with the latest build and see if this issue is resolved for you? Artifacts can be found here: https://github.com/DasAmpharos/EonTimer/actions/runs/8994382481

**Comment by DasAmpharos** [comment]
@valuta1995 can you try downloading the [EonTimer-Tools binaries](https://github.com/DasAmpharos/EonTimer-Tools/actions/runs/9026733867). Alternatively, you can run the EonTimer-Tools.py script from the [EonTimer-Tools repository](https://github.com/DasAmpharos/EonTimer-Tools).

Specifically, run the "Export Settings" action from the list of actions and share you settings here. Others have reported this issue on discord and I need some help resolving this issue.

**Comment by DasAmpharos** [comment]
@valuta1995 the issue should be fixed by #138

**Comment by DasAmpharos** [comment]
Others are reporting the issue to be resolved. Will go ahead and close this. You can still comment if you have any issues and I'll look into them
